src/services/AnalysisService.ts

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+type CommentMoodPayload = {
+  comment: string;
+}
+
+class AnalysisService {
+  private url: string;
+
+  constructor() {
+    this.url = process.env.AnalysisServerUrl!;
```

Comment:
let's pass it through ctor
so that we access all ENV variables in lambda initialization only